packages/block-editor/src/components/rich-text/index.js

```diff
@@ -39,6 +39,34 @@ import { useFormatTypes } from './use-format-types';
 import FormatEdit from './format-edit';
 import { getMultilineTag, getAllowedFormats } from './utils';
 
+/**
+ * Removes props used for the native version of RichText so that they are not
+ * passed to the DOM element and log warnings.
+ *
+ * @param {Object} props Props to filter.
+ *
+ * @return {Object} Filtered props.
+ */
+function removeNativeProps( props ) {
+	return omit( props, [
+		'__unstableMobileNoFocusOnMount',
+		'deleteEnter',
+		'placeholderTextColor',
+		'textAlign',
+		'selectionColor',
```

Comment:
Should a block be able to set selection color?

Reply:
I think we do still want this though as it enables a bit of control over color accessibility. The problem is that the selection highlight box and cursor can blend into the block's background color when we only use the platform's Native color choice. For the case of buttons, we have it set to match the [text color](https://github.com/WordPress/gutenberg/blob/fd41e1b0674bd928c342c653a6b15ed3ed16bf43/packages/block-library/src/button/edit.native.js#L471) as an attempt to provide some contrast. 

Here's the original issue that was opened for this to get some more context https://github.com/wordpress-mobile/gutenberg-mobile/issues/1773